### PR TITLE
Tokens::isLambda - fix detection near comments

### PR DESCRIPTION
**Pull request by keradus**
(After merge into 1.x) It should be manually merged into TokensAnalyzer at 2.x line.
Also watch on Token::getLine removal

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Changelog for v1.4
 ------------------
 
 * feature #841 PhpdocParamsFixer: added aligning var/type annotations (GrahamCampbell)
+* bug #950 Tokens::isLambda - fix detection near comments (keradus)
```
Comment (GrahamCampbell) [inline CHANGELOG.md]: 2.0?

Comment (keradus) [inline CHANGELOG.md]: 1.4?

Comment (GrahamCampbell) [inline CHANGELOG.md]: you've added an entry to the 1.4 change log and you've said you plan to merge this into 2.0. That doesn't make sense to me.

Comment (GrahamCampbell) [inline CHANGELOG.md]: ohh, i see what you meant now....

Comment (GrahamCampbell) [inline CHANGELOG.md]: lol, nvm then

**Comment by keradus** [comment]
merged

**Pull request merged**